update docs for async attach feature (#4753)
cypress-io/cypress-documentation: content/api/commands/visit.md

event.</li><li>`cy.visit()` can time out waiting for assertions you've added to
pass.</li></List>

> ## Visiting Cross Origin sites

Reviewer: ```suggestion
## Visiting cross-origin sites
```

<Alert type="warning">

<strong class="alert-header"><Icon name="exclamation-triangle"></Icon>
Experimental</strong>

> Visiting cross origin sites will currently throw an error, it can be enabled by

Reviewer 2: ```suggestion
Visiting cross origin sites will currently throw an error. It can be enabled by
```

Reviewer 2: is cross origin hyphenated?

> Visiting cross origin sites will currently throw an error. It can be enabled by

Reviewer: ```suggestion
Visiting cross-origin sites will currently throw an error. It can be enabled by
```

setting
the [`experimentalSessionAndOrigin`](/guides/references/experiments) flag
to `true` in the Cypress config. This will allow you to visit the cross origin

> site but to interact with the cross origin site you must use a

Reviewer 2: ```suggestion
site without errors, however, to interact with the content on the cross origin site you must use a
```

> the [`experimentalSessionAndOrigin`](/guides/references/experiments) flag
> to `true` in the Cypress config. This will allow you to visit the cross origin
> site without errors, however, to interact with the content on the cross origin site you must use a
> [`cy.origin()`](/api/commands/origin) block.

Reviewer: ```suggestion
the [`experimentalSessionAndOrigin`](/guides/references/experiments) flag
to `true` in the Cypress config. This will allow you to visit the cross origin
site without errors. However, to interact with the content on the cross origin site, you must use a
[`cy.origin()`](/api/commands/origin) block.
```

When visiting a cross origin site, the `onBeforeLoad` and `onLoad` options are

> disallowed.

Reviewer 2: ```suggestion
not supported.
```

> When visiting a cross origin site, the `onBeforeLoad` and `onLoad` options are

Reviewer: ```suggestion
When visiting a cross-origin site, the `onBeforeLoad` and `onLoad` options are
```
